### ISSUE TITLE
"indent": [2, "tab"], is flagging properly formatted code

### ISSUE THREAD

**Issue by ChetHarrison** [eslint/eslint#1927]
for example it flags the `'use strict'` here

```javascript
(function() {
    'use strict';

})();
```

**Comment by gyandeeps** [comment]
I tried this code in http://eslint.org/demo/ and it worked fine for me.
When I coped the code from here it had spaces but then when I put `tabs` it showed lint free.

**Comment by ChetHarrison** [comment]
Sorry my fault my editor was set to spaces not tabs.
